src/main/kotlin/fi/metatavu/timebank/api/impl/translate/UserTranslator.kt

```diff
@@ -0,0 +1,44 @@
+package fi.metatavu.timebank.api.impl.translate
+
+import fi.metatavu.timebank.api.forecast.models.ForecastPerson
+import fi.metatavu.timebank.api.keycloak.KeycloakController
+import fi.metatavu.timebank.model.Person
+import fi.metatavu.timebank.model.User
+import org.keycloak.admin.client.resource.UsersResource
+import org.keycloak.representations.idm.UserRepresentation
+import java.util.*
+import javax.enterprise.context.ApplicationScoped
+import javax.inject.Inject
+
+@ApplicationScoped
+class UserTranslator: AbstractTranslator<UserRepresentation, User>() {
+    @Inject
+    lateinit var keycloakController: KeycloakController
+
+    override fun translate(entity: UserRepresentation): User {
+        val keycloakUser = keycloakController.findUserByEmail(entity.email)
```

Comment:
This seems like a weird setup. Why would you want to re-find te user in this case?

Why not just 
```kotlin
    override fun translate(entity: UserRepresentation): User {
        return User(
            id = UUID.fromString(entity.id),
            ...
    }
```